searx/webapp.py

```diff
@@ -585,15 +586,14 @@ def post_request(response):
         'render;dur=' + str(round(request.render_time * 1000, 3)),
     ]
     if len(request.timings) > 0:
-        timings = sorted(request.timings, key=lambda v: v['total'])
+        timings = sorted(request.timings, key=lambda t: t.total)
```

Comment:
Nit: Is the renaming "v --> t" of this lambda needed? 

IMO it is better we have 'v' in the lambda and 't' in the for loops below.

Reply:
I like to name lambda parameters after their type so I think `t` for a `Timing` makes sense.